README.rst

```diff
@@ -182,30 +182,30 @@ Fedora / CentOS (rpm)
 Comparison to related technologies
 ==================================
 
-#. *Data Engineering* tools such as `AirFlow <https://airflow.apache.org/>`_, `Luigi <https://github.com/spotify/luigi>`_ and
-   others - in DVC data, model and ML pipelines represent a single ML project and focuses on data scientist' experience while
-   data engineering pipeline orchestrates multiple data projects and focuses on efficient execution. DVC project can be used
-   from data pipeline as a single execution step. `AirFlow DVC <https://github.com/covid-genomics/airflow-dvc>`_ is an example
-   of such integration.
+#. Data Engineering tools such as `AirFlow <https://airflow.apache.org/>`, `Luigi <https://github.com/spotify/luigi>`, and
+   others - in DVC data, model and ML pipelines represent a single ML project focused on data scientists' experience.
+   Data engineering tools orchestrate multiple data projects and focus on efficient execution. A DVC project can be used
+   from existing data pipelines as a single execution step.
 
 #. `Git-annex <https://git-annex.branchable.com/>`_ - DVC uses the idea of storing the content of large files (which should
    not be in a Git repository) in a local key-value store, and uses file hardlinks/symlinks instead of
    copying/duplicating files.
 
-#. `Git-LFS <https://git-lfs.github.com/>`_ - DVC is compatible with any remote storage (S3, Google Cloud, Azure, SSH,
+#. `Git-LFS <https://git-lfs.github.com/>`_ - DVC is compatible with many remote storage services (S3, Google Cloud, Azure, SSH,
    etc). DVC also uses reflinks or hardlinks to avoid copy operations on checkouts; thus handling large data files
    much more efficiently.
 
-#. *Makefile* (and analogues including ad-hoc scripts) - DVC tracks dependencies (in a directed acyclic graph).
+#. Makefile (and analogues including ad-hoc scripts) - DVC tracks dependencies (in a directed acyclic graph).
 
 #. `Workflow Management Systems <https://en.wikipedia.org/wiki/Workflow_management_system>`_ - DVC is a workflow
    management system designed specifically to manage machine learning experiments. DVC is built on top of Git.
 
-#. `DAGsHub <https://dagshub.com/>`_ - online service of Git+DVC repositories with pipeline and metrics visualization and
-   DVC-specific cloud storage.
+#. `DAGsHub <https://dagshub.com/>`_ - An online service that integrates Git & DVC. Used to version and catalog data and models, 
+   track experiments, visualize pipelines. 
```

Comment:
I'm not very sure about this change. DVC already integrates with Git for example. I know DAGsHub has evolved since this was written but I'm not too familiar cc @efiop

Reply:
Other than that maybe the text needs wrapping to 80 or 120 chars?

Reply:
I took that DAGsHub content their website, but feel free to change or let me know if further changes required from me. Thanks for the edits.

Reply:
I've updated this bit. Maybe you can wrap the contents to 80 characters while someone else approves the changes? I think that's what's being used here. Thanks again @himanshugarg

Reply:
Wrapped modified sections to 80 chars but happy to wrap the entire file if review won't won't be a problem.  Thanks